app/src/createExcelFiles/ExcelUtils.js

```diff
@@ -0,0 +1,23 @@
+import * as XLSX from "xlsx";
+
+export function mapToExcelArray(searchItems) {
+  const result = searchItems.map((searchItem) => [
+    searchItem.title,
+    searchItem.link,
+  ]);
+  result.unshift(["Title", "Links"]);
+  return result;
+}
+
+export function createExcel(googleResults, searchName) {
+  const arrayOfArray = mapToExcelArray(googleResults);
```

Comment:
your variable naming is REALLY off. You need to CONSCIOUSLY be naming these variables.
What the heck is the input `googleResults` and `searchName` ? 
it should be more like...`searchItemList` and `searchQuery`

What is `const arrayOfArrays` ?? perhaps something like excelResultList or something would be better ??